src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftTimedStepDescriptor.java

```diff
@@ -0,0 +1,62 @@
+package com.openshift.jenkins.plugins.pipeline.model;
+
+import hudson.util.ListBoxModel;
+
+public interface IOpenShiftTimedStepDescriptor {
+
+	// TODO: Use an enum or Time type to standardize units, perhaps a constant
+
+	String getWait();
+
+	void setWait(String wait);
+
+	String getWaitUnit();
+
+	void setWaitUnit(String waitUnit);
+
+	default void doFillWaitUnit(String defaultWaitUnit) {
```

Comment:
i think the dsl versions that i've called for being moved to this interface are different in signature than these existing methods, but that said, there could be the opportunity to share some logic - just keep that in mind